### DISCUSSION TITLE
What license to use for open source projects at UHC?

### DISCUSSION THREAD

**Discussion by ran-codes** [Drexel-UHC/analytics-corner#8] [Q&A]
@alinasmahl1 I think @GraceSigalla mentioned you guys were using some license ([MIT ](https://github.com/twbs/bootstrap/blob/v5.0.2/LICENSE)or [GNU GPL v2](https://github.com/jgm/pandoc/blob/master/COPYING.md) or something else I forgot) for one of your repositories. I want to put a license on some public repositories at the [Drexeh-UHC](https://github.com/Drexel-UHC) organization. Any suggestion for which license to use? (feel free to tag @ anyone else who may know more about this stuff).

**Comment by alinasmahl1** [comment]
@ran-codes I think I've used MIT in the past, but just as a random license, not for any specific reason. I thin kusing a Drexel-UHC license makes sense, I just really havne't used licenses in an informed way to date, so can't really be helpful

**Reply by ran-codes** [reply]
got it! yea I'll just stick with MIT for now. maybe @usamabilal can share thoughts when he gets back.
